tests/test_taxi.py

```diff
@@ -12,9 +12,7 @@ def setUp(self) -> None:
         )
         self.client.force_login(self.admin_user)
         self.driver = get_user_model().objects.create_user(
```

Comment:
Do not modify files in tests folder

Reply:
I think it did the black formatter. I will try to roll back